src/pyHexDump/cmd_print.py

```diff
@@ -436,6 +441,16 @@ def _print_template(binary_data, cfg_elements_dict, template):
     """
     ret_status = Ret.OK
     element_value_dict = _get_element_value_dict(binary_data, cfg_elements_dict)
+
+    #Add methods and the binary_data to the element_value_dict to execute them from the template
+    element_value_dict["makros_compare_values"] = makros_compare_values
+    element_value_dict["makros_check_stadabm"] = makros_check_stadabm
+    element_value_dict["mem_access_get_api_by_data_type"] = mem_access_get_api_by_data_type
+    element_value_dict["calc_checksum"] = calc_checksum
+    element_value_dict["convert_middle_to_little_endian"] = convert_middle_to_little_endian
+    element_value_dict["binary_data"] = binary_data
+    element_value_dict["bootloader_start_addr"] = 0x000000

```

Comment:
@steltobs pyHexDump shall be generic and not contain specific stuff, only necessary for a user specific report. Therefore I removed some macros. Please put them to your template.
Please consider the new list of macros in your template. See chapter for macros in the README.